.github/workflows/preview.yml

```diff
@@ -1,45 +1,78 @@
 name: Preview
 on:
   pull_request:
-    branches: 
+    branches:
       - master
       - develop
-      - 'release/**'
+      - "release/**"
 jobs:
   deployment:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      
-      - uses: aws-actions/configure-aws-credentials@v1
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: cofigure aws
+        uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ secrets.AWS_REGION }}
-      
-      - name: install deps
+
+      - name: set $SHA7
+        run: echo ::set-env name=SHA7::$(git rev-parse --short ${{ github.event.pull_request.head.sha || github.sha }})
+
+      - name: set $PREVIEW_URL
+        run: echo ::set-env name=PREVIEW_URL::"https://"$SHA7".ods.dev"
+
+      - name: get COMMIT_MSG
+        run: |
+          MSG=$(git log --format=%B -n 1 ${{ github.event.after }})
+          echo "::set-env name=COMMIT_MSG::${MSG}"
+
+      # - name: Dump GitHub context
+      #   env:
+      #     GITHUB_CONTEXT: ${{ toJson(github) }}
+      #   run: echo "$GITHUB_CONTEXT"
+
+      - name: install dependencies
         run: yarn
-      
+
       - name: test
         run: ./node_modules/lerna/cli.js run test
-        
+
       - name: build docs
         run: yarn workspace @okta/design-docs build
 
-      - name: deploying docs to ods.dev
-        run: | 
-          SHA=$(git rev-parse --short ${{ github.event.pull_request.head.sha || github.sha }})
-          aws s3 sync ./packages/docs/dist/ s3://ods.dev/$SHA --delete
-      
-      - name: deployment complete
+      - name: "deploy: create"
+        uses: bobheadxi/deployments@master
```

Comment:
Do not rely on the `master` version here. Please pin to a [release version](https://github.com/bobheadxi/deployments/releases/tag/v0.4.2) to avoid unexpected changes.